components/Avatar/avatar.css

```diff
@@ -0,0 +1,60 @@
+.avatars-wrapper {
+	align-items: baseline;
+	align-self: center;
+	background-color: var(--white);
+	display: flex;
+	justify-content: center;
+	justify-self: center;
+	margin: 1rem;
+	padding: 1rem;
+	width: 80%;
+}
+.avatar {
+	height: 5rem;
+	margin: 1rem;
+	object-fit: cover;
+	width: 5rem;
+}
+.avatar-img {
```

Comment:
Avatar image looks a bit distorted. You can fix this using `object-fit: cover`

Reply:
Thank you for the suggestion. Fixed it. Images look much better now.